.github/workflows/cd.yml

```diff
@@ -0,0 +1,44 @@
+name: Dependabot CD to GCP
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy-dependabot-merge-action-app:
+    name: Build & Deploy dependabot-merge-action-app
+    runs-on: ubuntu-latest
+
+    env:
+      PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
+      CLOUDRUN_SERVICE_NAME: dependabot-merge-action-app
+      CLOUDRUN_SERVICE_REGION: us-central1
+      API_ID_VALUE: ${{ secrets.API_ID }}
+      PRIVATE_KEY_VALUE: ${{ secrets.PRIVATE_KEY}}
+
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Set up gcloud CLI
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: latest
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
```

Comment:
we need documentation (it can be a section in the README) about the steps necessary to deploy the application, including:

- prerequisites (e.g. a GCP account)
- how to create the service account needed to execute the deployment
- which permissions the service account needs (include a link to the official docs but specify the roles that we know we need)
- anything else...

Reply:
Created the documentation to deploy the app